eternalcore-core/src/main/java/com/eternalcode/core/translation/implementation/PLTranslation.java

```diff
@@ -602,6 +607,7 @@ public static class PLInventorySection implements InventorySection {
     @Description({
         " ",
         "# Ta sekcja odpowiada za interakcję z graczami za pomocą komend",
+        "# Ta sekcja odpowiada za interakcję z graczami za pomocą komend",
```

Comment:
![image](https://github.com/user-attachments/assets/a1ce919c-94b9-49ed-8919-a993e3d5e62e)

Reply:
What do you mean?

Reply:
Duplicated, review ai says same. https://github.com/EternalCodeTeam/EternalCore/pull/856#discussion_r1792402044